webapp/sources/rudder/rudder-web/src/main/gulpfile.mjs

```diff
@@ -133,7 +149,7 @@ function vendor_js(cb) {
 };
 
 function css(cb) {
-    src(paths.css.src)
+    src(paths.css.src, { encoding: false })
```

Comment:
I had to add this option to make ionicons and fontawesome work.

Gulp 5 stream encoding is now set to UTF-8 by default, which is not compatible with these icon libs :

> Our streams now default to UTF-8 encoding. Previously, our streams took whatever data was emitted without considering any encoding. However, in this release, we resolved a 10 year old [issue](https://github.com/gulpjs/vinyl-fs/issues/23) to support custom encodings and we default these to UTF-8. Most usage of gulp won’t need to change anything, but certain plugins may produce non-UTF-8 output and you’ll need to set `{ encoding: false }` on your gulp streams.